### ISSUE TITLE
Try to use VXI for VXML

### ISSUE THREAD

**Issue by cgueret** [WorldWideSemanticWeb/KasaDaka#2]
http://downloads.i6net.com/vxi/raspbian/
http://www.voip-info.org/wiki/index.php?page_id=4042

**Comment by cgueret** [comment]
Comment from Stephane: " the issue is that vxi is just the library you have an enormous amount of work to do to use openVXI to make a voicexml browser. this is exactly what voiceglue did. it is good to know that open vxi has been ported to raspbian, this is a key element for vg."

**Comment by abaart** [comment]
Working! See my fork

**Comment by cgueret** [comment]
Awesome news! Thx a bunch for all the work. Shall we merge your fork back with the master branch ?

**Comment by abaart** [comment]
Yes, but maybe wait until everything is approved and final! Then I will report back!
